manager/controllers/baseline_systems.go

```diff
@@ -28,13 +28,17 @@ type BaselineSystemsDBLookup struct {
 }
 
 type BaselineSystemAttributes struct {
-	DisplayName string `json:"display_name" csv:"display_name" query:"sp.display_name" gorm:"column:display_name"`
+	// Baseline system display name
+	DisplayName string `json:"display_name" csv:"display_name" query:"sp.display_name" gorm:"column:display_name" example:"my-baselined-system"` // nolint: lll
 }
 
 type BaselineSystemItem struct {
+	// Additional baseline system attributes
 	Attributes BaselineSystemAttributes `json:"attributes"`
-	ID         string                   `json:"id"`
-	Type       string                   `json:"type"`
+	// Baseline system inventory ID (uuid format)
+	InventoryID string `json:"inventory_id" example:"00000000-0000-0000-0000-000000000001"`
```

Comment:
@mkholjuraev I've changed this field ID -> InventoryID to be consistent with the rest of the API. Don't you mind? Maybe it will impact baseline API usage so you will need to update frontend code accordingly.

Reply:
I do not mind. This looks more consistent.